README.md

```diff
@@ -1,5 +1,14 @@
-# 📗 Table of Contents
```

Comment:
- [ ] The table of content should look like the one from the template provided for use by Microverse, kindly fix that.
- [ ] I was expecting to find a license link under the license header, but there is nothing, kindly fix this issue and make the link to work properly without any error.